presto-docs/src/main/sphinx/admin/cte-materialization.rst

```diff
@@ -0,0 +1,103 @@
+=================================
+CTE Materialization
+=================================
+
+CTEs (Common Table Expressions) are defined as subqueries that appear in a WITH clause provided by the user.
+Their repeated usage within a query can lead in unnecessary redundant computations, excessive data retrieval, and high resource consumption
+
+To address this, Presto supports CTE Materialization. This feature allows intermediate CTEs to be materialized and effectively stored
```

Comment:
"materialized and effectively stored
and reused within" --> materialized, stored, and reused within

or maybe just "reused"